typespec-extension/src/code-model-builder.ts

```diff
@@ -213,8 +214,8 @@ export class CodeModelBuilder {
     this.sdkContext = createSdkContext(context, "@azure-tools/typespec-java");
     if (this.supportsAdvancedVersioning()) {
       this.sdkContext.apiVersion = "all";
-    } else {
-      this.sdkContext.apiVersion = "latest";
+    } else if (this.options["api-version"]) {
+      this.sdkContext.apiVersion = this.options["api-version"];
```

Comment:
Could it be "latest" or "all" here?

Reply:
Yeah, it'll read from emitter-options. Removed.